plugins/plugin-svn/che-plugin-svn-ext-ide/src/main/java/org/eclipse/che/plugin/svn/ide/SubversionExtensionLocalizationConstants.java

```diff
@@ -0,0 +1,541 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.plugin.svn.ide;
+
+import com.google.gwt.i18n.client.Messages;
+
+/**
+ * Interface providing access to localization constants.
+ * <p/>
+ * These are loaded from
+ * src/resources/org/eclipse/che/plugin/svn/SubversionExtensionLocalizationConstants.properties
```

Comment:
Remove or rewrite this outdated comment. For now it locates in package org/eclipse/che/plugin/svn/ide
I prefer to remove it.